data/registry/instrumentation-java-okhttp.yml

```diff
@@ -13,4 +13,8 @@ authors:
 urls:
   repo: https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/okhttp
 createdAt: 2020-11-05
+package:
+  registry: maven
+  name: io.opentelemetry.instrumentation/opentelemetry-okhttp-3.0
```

Comment:
you can fix that spelling issue by adding `# cspell:ignore okhttp` to the top of the file, see how other files have it:

https://github.com/open-telemetry/opentelemetry.io/blob/main/data/registry/application-integration-python-checkmk.yml